utils/fetch.js

```diff
@@ -6,7 +6,7 @@ export const fetchCampaigns = async () => {
 
   try {
     const { data } = await axios.get(
-      `${process.env.NEXT_PUBLIC_CROWDFUNDING_API}/campaigns?page=0&size=20`,
+      `${process.env.NEXT_PUBLIC_CROWDFUNDING_API}/campaigns?page=0&size=20&search=`,
```

Comment:
Sorry but why is `search=` needed? dont understand if a parameter is not passed

Reply:
This is a BE bug that is fixed in an open PR, I'm going to delete it, it shouldn't be that &search="" I will delete it.